core/src/main/scala/org/broadinstitute/dsde/rawls/entities/local/LocalEntityProvider.scala

```diff
@@ -18,9 +18,15 @@ class LocalEntityProvider(workspace: Workspace, dataSource: SlickDataSource)
 
   import dataSource.dataAccess.driver.api._
 
-  override def createEntity(entity: Entity): Future[Entity] = {
-    val workspaceContext = SlickWorkspaceContext(workspace)
+  private val workspaceContext = SlickWorkspaceContext(workspace)
+
+  override def entityTypeMetadata(): Future[Map[String, EntityTypeMetadata]] = {
+    dataSource.inTransaction { dataAccess =>
+      dataAccess.entityQuery.getEntityTypeMetadata(workspaceContext)
+    }
+  }
```

Comment:
LocalEntityProvider doesn't have any unit tests

Reply:
probably worth it to add unit tests; `EntityApiServiceSpec` tests all the features in here, though